pycbc/types/timeseries.py

```diff
@@ -243,18 +243,20 @@ def get_sample_times(self):
 
     def at_time(self, time, nearest_sample=False,
                 interpolate=None, extrapolate=None):
-        """ Return the value at the specified gps time
+        """ Return the value of the TimeSeries at the specified GPS time.
 
         Parameters
         ----------
+        time: scalar or array-like
```

Comment:
Should it be immediately clear to the user/reader which of these a LIGOTimeGPS object should be? 
If not, state this here (eg 'LIGOTimeGPS counts as a scalar')